source/MaterialXContrib/MaterialXNode/MaterialXShadingNodeImpl.cpp

```diff
@@ -74,15 +75,13 @@ MStatus bindFileTexture(MHWRender::MShaderInstance& shader,
     MStatus status = MStatus::kFailure;
 
     // Bind file texture
-    MaterialX::FilePath imagePath = searchPath.find(fileName);
-    bool imageFound = imagePath.exists();
-    if (imageFound)
+    MHWRender::MRenderer* renderer = MHWRender::MRenderer::theRenderer();
+    MHWRender::MTextureManager* textureManager = renderer ? textureManager = renderer->getTextureManager() : nullptr;
+    if (textureManager)
     {
-        MHWRender::MRenderer* renderer = MHWRender::MRenderer::theRenderer();
-        if (renderer)
+        MaterialX::FilePath imagePath = MaterialXMaya::findInSubdirectories(searchPath, fileName);
```

Comment:
Use new tree search utility. We can have images anywhere under the search roots.